tf_agents/agents/ddpg/critic_network.py

```diff
@@ -28,6 +29,9 @@ class CriticNetwork(network.Network):
 
   def __init__(self,
                input_tensor_spec,
+               preprocessing_layers=None,
+               preprocessing_combiner=None,
+               batch_squash=True,
```

Comment:
why do we need this argument publicly available?

Reply:
This arguments will pass to encoding_network.EncodingNetwork create an EncodingNetwork  for combi observations and using custom observation preprocessing layers such as Attention.

Reply:
do you ever need to pass a value other than `batch_squash=True` to EncodingNetwork?

Reply:
@rk37 no problem